### ISSUE TITLE
Crash with Plated

### ISSUE THREAD

**Issue by Dragon-Seeker** [Eltrut-Co/Lepton-Fabric#1]
Heres the paste bin for the crash:
https://pastebin.com/4XeSQKkh

The main top argument is:
java.lang.IllegalArgumentException: Cannot get property class_2746{name=waterlogged, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{lepton:polished_andesite_pressure_plate}

**Comment by Steven-Baez** [comment]
Sorry, I can't reproduce this but maybe someone else can, are you using any other mods that might do this?

**Comment by Dragon-Seeker** [comment]
Used the most current one uploaded to Curse. Ill check if the file got corrupted by redownloading.

**Comment by Steven-Baez** [comment]
ok

**Comment by Dragon-Seeker** [comment]
For more information its caused when loading a world that has been pregenerated or newly generated.

**Comment by StevePlayzz** [comment]
that didn't have the mod before?

**Comment by Dragon-Seeker** [comment]
I'm going to double check if its mod incompatibility or a main mod issue as re downloading didn't fix such.

**Comment by Steven-Baez** [comment]
Yeah, it might be that because I tried to load a world or make a new one and it still doesn't crash

**Comment by Dragon-Seeker** [comment]
Ok so by it self, it works but some mod is causing this issue so ill attempt to find out which.

**Comment by Steven-Baez** [comment]
ok nice

**Comment by Dragon-Seeker** [comment]
Its seems to be an incompatibility with a mod named [Plated](https://www.curseforge.com/minecraft/mc-mods/plated). It allows the placement of pressure plates to be placed on walls.

**Comment by Steven-Baez** [comment]
Ah, I see, it might be because we have our own pressure plate class and that's what breaking it, I have to see if I can fix that

**Comment by Steven-Baez** [comment]
Sorry but I can't fix it because Minecraft has protected access to the pressure plate class, so you're going to have to make an issues to plated, to see if they can fix it :(

**Comment by Dragon-Seeker** [comment]
K, ill see about saying something but could you at least give a warning for anyone who might use that mod with lepton?

**Comment by Steven-Baez** [comment]
sure I'll keep this open and but something in the front page
